data/strings/strings.txt

```diff
@@ -16326,6 +16326,234 @@
     zh-Hans = 3. 使用应用时选择
     zh-Hant = 3. 使用應用時選擇
 
+  [location_services_disabled_on_device_1]
```

Comment:
How is this string different from the location_services_disabled_1 string above?

Reply:
They are the same for now. Is it possible to use the same value for different keys in localized strings without duplicating values? I duplicate them to change separately if needed.

Reply:
What is the value of duplicating existing strings? Let's avoid preliminary optimization. If someone needs to add new, different strings, for another task, then he/she will make a copy of existing strings and modify them.

Reply:
Already fixed!